frontend/package.json

```diff
@@ -92,6 +92,7 @@
     "spacetime": "^7.4.0",
     "stream-browserify": "^3.0.0",
     "stream-http": "^3.2.0",
+    "tar": "^7.4.3",
```

Comment:
tar is only missing from the plugins/headlamp-plugin, no need to add to the frontend